src/script/ismine.cpp

```diff
@@ -49,9 +49,9 @@ isminetype IsMineInner(const CKeyStore& keystore, const CScript& scriptPubKey, b
     std::vector<valtype> vSolutions;
     txnouttype whichType;
     if (!Solver(scriptPubKey, whichType, vSolutions)) {
-        if (keystore.HaveWatchOnly(scriptPubKey))
-            return ISMINE_WATCH_UNSOLVABLE;
-        return ISMINE_NO;
+        if (keystore.HaveWatchOnly(scriptPubKey)) {
```

Comment:
commit: Make IsMine stop distinguishing solvable/unsolvable

Does this even need to be here? Wouldn't it just do the watchonly check at the end of the function anyway?

EDIT: I see it is dropped in the next commit anyway.